src/components/progress-report/migrations/UpdateExplanationReasons.migration.ts

```diff
@@ -0,0 +1,42 @@
+import { node, relation } from 'cypher-query-builder';
+import { BaseMigration, Migration } from '~/core';
+import { ACTIVE } from '~/core/database/query';
+
+@Migration('2023-03-10T17:00:00')
+export class UpdateExplanationReasonsMigration extends BaseMigration {
+  private readonly findAndReplaceReason = async (
```

Comment:
```suggestion
  private async findAndReplaceReason(
```

Reply:
The linter was crying about needing it to be readonly. That's my I put it there